packages/react-components/src/components/chart/utils/handleSyncTrendCursors.ts

```diff
@@ -0,0 +1,47 @@
+import { SyncChanges } from '../types';
+
+// This handles the sync between the Zustand sync state and local state
+// a user action can be one of the following
+// add , delete or drag. given only one can happen at a given time returning as soon as we find that an action has occurred
+export const handleSyncChanges = ({ syncedTrendCursors = {}, graphic }: SyncChanges) => {
+  const ExistingIds: string[] = graphic.map((g) => g.id as string);
```

Comment:
Nit: `ExistingIds` instead of `existingIds`?

Reply:
done